src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -377,6 +429,76 @@ private void basicAck(Channel channel, long deliveryTag) throws IOException {
         channel.basicAck(deliveryTag, false);
     }
 
+    public void lockSendingIfSizeLimitExceeded(String routingKey) {
+        lockSendingIfSizeLimitExceeded(List.of(routingKey));
+    }
+
+    private void lockSendingIfSizeLimitExceeded() {
+        lockSendingIfSizeLimitExceeded(
+                channelsByPin.keySet().stream()
+                        .filter(channelHolder -> channelHolder.routingKey != null)
+                        .map(channelHolder -> channelHolder.routingKey)
+                        .collect(Collectors.toList())
```

Comment:
The result of this expression is constant. Please build this value according to mq.json config

Reply:
done